src/SqlStreamStore.Postgres/PgSqlScripts/Migration.sql

```diff
@@ -1,329 +1,22 @@
-  THEN
-    _created_utc = now() at time zone 'utc';
-  END IF;
-  SELECT __schema__.streams.id_internal
-      INTO _stream_id_internal
-  FROM __schema__.streams
-  WHERE __schema__.streams.id = _stream_id;
-
-  IF _expected_version = -1 /* ExpectedVersion.NoStream */
-  THEN
-    RAISE EXCEPTION 'WrongExpectedVersion';
-  ELSIF _expected_version >= 0 /* ExpectedVersion */
-    THEN
-      IF _stream_id_internal IS NULL
-      THEN
-        RAISE EXCEPTION 'WrongExpectedVersion';
-      END IF;
-
-      SELECT __schema__.messages.stream_version
-          INTO _latest_stream_version
-      FROM __schema__.messages
-      WHERE __schema__.messages.stream_id_internal = _stream_id_internal
-      ORDER BY __schema__.messages.position DESC
-      LIMIT 1;
-
-      IF _latest_stream_version != _expected_version
-      THEN
-        RAISE EXCEPTION 'WrongExpectedVersion';
-      END IF;
-  END IF;
-
-  DELETE FROM __schema__.messages WHERE __schema__.messages.stream_id_internal = _stream_id_internal;
-
-  DELETE FROM __schema__.streams WHERE __schema__.streams.id = _stream_id;
-
-  IF (_deletion_tracking_disabled = TRUE)
-      THEN
-      RETURN;
-  END IF;
-
-  GET DIAGNOSTICS _affected = ROW_COUNT;
-
-  IF _affected > 0
-  THEN
-    PERFORM __schema__.append_to_stream(
-              _deleted_stream_id,
-              _deleted_stream_id_original,
-              NULL,
-              -2,
-              _created_utc,
-              ARRAY [_deleted_stream_message]);
-
-  END IF;
-END;
-
-$F$
-LANGUAGE 'plpgsql';
-
-CREATE OR REPLACE FUNCTION __schema__.delete_stream_messages(
-  _stream_id                  CHAR(42),
-  _message_ids                UUID [],
-  _deletion_tracking_disabled BOOLEAN,
-  _deleted_stream_id          CHAR(42),
-  _deleted_stream_id_original VARCHAR(1000),
-  _created_utc                TIMESTAMP WITH TIME ZONE,
-  _deleted_messages           __schema__.new_stream_message []
-)
-  RETURNS VOID
-AS $F$
-DECLARE
-  _stream_id_internal INT;
-  _deleted_count      NUMERIC;
-BEGIN
-  IF _created_utc IS NULL THEN
-    _created_utc = now() at time zone 'utc';
-  END IF;
-
-  SELECT __schema__.streams.id_internal
-      INTO _stream_id_internal
-  FROM __schema__.streams
-  WHERE __schema__.streams.id = _stream_id;
-
-  WITH deleted AS (DELETE FROM __schema__.messages
-  WHERE __schema__.messages.stream_id_internal = _stream_id_internal
-        AND __schema__.messages.message_id = ANY (_message_ids)
-  RETURNING *)
-  SELECT count(*)
-  FROM deleted
-      INTO _deleted_count;
-
-  IF (_deletion_tracking_disabled = FALSE AND _deleted_count > 0) 
-  THEN
-    PERFORM __schema__.append_to_stream(
-              _deleted_stream_id,
-              _deleted_stream_id_original,
-              NULL,
-              -2,
-              _created_utc,
-              _deleted_messages);
-  END IF;
-END;
-
-$F$
-LANGUAGE 'plpgsql';
+COMMENT ON SCHEMA __schema__ IS '{ "version": 3 }';
 
 DROP FUNCTION public.read_all(int4, int8, bool, bool);
```

Comment:
This would remove an in active function in prod with our idea.
Wouldn't do this